cli.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+'use strict';
+
```

Comment:
I would like to modularize this code a little bit.
What you think about creating those 3 functions:
help - That just prints the usage - See [this](https://github.com/simonepri/pidtree/blob/master/bin/pidtree.js#L8)
get - That gets the osmid and prints the geojson to stdout
run - That parses the cli arguments - See [this](https://github.com/simonepri/pidtree/blob/master/bin/pidtree.js#L95)